バグ：SQS接続不可時のエラーハンドリング

### 現在の問題
・SQSに接続出来なかった時のエラーハンドリングが出来ていない。

```
 [ERROR] Error handling request /v1/Omise/Charge
Traceback (most recent call last):
  File "/app/tmr-node/app/api/v1/omise.py", line 323, in send_sqs_msg
    queue = sqs.get_queue_by_name(QueueName=name)
  File "/home/apl/.pyenv/versions/3.6.2/lib/python3.6/site-packages/boto3/resources/factory.py", line 520, in do_action
    response = action(self, *args, **kwargs)
  File "/home/apl/.pyenv/versions/3.6.2/lib/python3.6/site-packages/boto3/resources/action.py", line 83, in __call__
    response = getattr(parent.meta.client, operation_name)(**params)
  File "/home/apl/.pyenv/versions/3.6.2/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/apl/.pyenv/versions/3.6.2/lib/python3.6/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.QueueDoesNotExist: An error occurred (AWS.SimpleQueueService.NonExistentQueue) when calling the GetQueueUrl operation: The specified queue does not exist or you do not have access to it.
```

### 修正の方針
検討中

--- a/app/api/v1/omise.py
+++ b/app/api/v1/omise.py
@@ -173,7 +173,6 @@ class Charge(BaseResource):
         customer_id = request_json['customer_id']
         amount = request_json['amount']
 
-        '''
         # Card情報を取得
         try:
             customer = omise.Customer.retrieve(customer_id)
@@ -243,7 +242,6 @@ class Charge(BaseResource):
             # Charge状態を[ERROR]ステータスに更新する
             omise_charge.status = OmiseChargeStatus.ERROR.value
             raise InvalidCardError
-        '''
 
         sqs_msg = {
             'exchange_address': exchange_address,
@@ -259,7 +257,7 @@ class Charge(BaseResource):
             raise AppError
 
         # 全ての処理が正常処理された場合、Charge状態を[SUCCESS]ステータスに更新する
-        #omise_charge.status = OmiseChargeStatus.SUCCESS.value
+        omise_charge.status = OmiseChargeStatus.SUCCESS.value
 
         self.on_success(res)
 

